Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/UserInterface.java

```diff
@@ -0,0 +1,24 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+public interface UserInterface {
+
+	public String playerName();
```

Comment:
I don't think it should be the responsiblity of UserInterface to deal with players. Hold it in the game logic or have a PlayersList or something.